Server/src/index.ts

```diff
@@ -76,346 +76,355 @@ export class CitrineOSServer {
+      );
     }
-    this._config = this.configService?.systemConfig as SystemConfig
+    this._config = this.configService?.systemConfig as SystemConfig;
 
     // Create server instance
-    this._server = fastify().withTypeProvider<JsonSchemaToTsProvider>()
+    this._server = fastify().withTypeProvider<JsonSchemaToTsProvider>();
 
     // Add health check
-    this.initHealthCheck()
+    this.initHealthCheck();
 
     // Create Ajv JSON schema validator instance
-    this._ajv = this.initAjv()
-    this.addAjvFormats()
-
-    // Initialize parent logger
-    this._logger = this.initLogger()
+    this._ajv = this.initAjv();
+    this.addAjvFormats();
 
     // Force sync database
-    this.forceDbSync()
+    this.forceDbSync();
 
     // Initialize Swagger if enabled
-    this.initSwagger()
+    this.initSwagger();
 
     // Add Directus Message API flow creation if enabled
     if (this._config.util.directus?.generateFlows) {
-      const directusUtil = new DirectusUtil(this._config, this._logger)
+      const directusUtil = new DirectusUtil(
+        this._config,
+        this.loggerService?.logger
+      );
       this._server.addHook(
-        'onRoute',
+        "onRoute",
         directusUtil.addDirectusMessageApiFlowsFastifyRouteHook.bind(
           directusUtil
         )
-      )
-      this._server.addHook('onReady', async () => {
-        this._logger?.info('Directus actions initialization finished')
-      })
+      );
+      this._server.addHook("onReady", async () => {
+        this.loggerService?.logger?.info(
+          "Directus actions initialization finished"
+        );
+      });
     }
 
     // Register AJV for schema validation
-    this.registerAjv()
+    this.registerAjv();
 
     // Initialize module & API
     // Always initialize API after SwaggerUI
-    this.initSystem(appName)
+    this.initSystem(appName);
 
-    process.on('SIGINT', this.shutdown.bind(this))
-    process.on('SIGTERM', this.shutdown.bind(this))
-    process.on('SIGQUIT', this.shutdown.bind(this))
+    process.on("SIGINT", this.shutdown.bind(this));
+    process.on("SIGTERM", this.shutdown.bind(this));
+    process.on("SIGQUIT", this.shutdown.bind(this));
   }
 
-  private initHealthCheck () {
-    this._server.get('/health', async () => {
-      return { status: 'healthy' }
-    })
+  private initHealthCheck() {
+    this._server.get("/health", async () => {
+      return { status: "healthy" };
+    });
   }
 
-  private initAjv (ajv?: Ajv) {
+  private initAjv(ajv?: Ajv) {
     return (
       ajv ||
       new Ajv({
-        removeAdditional: 'all',
+        removeAdditional: "all",
         useDefaults: true,
-        coerceTypes: 'array',
-        strict: false
+        coerceTypes: "array",
+        strict: false,
       })
-    )
+    );
   }
 
-  private addAjvFormats () {
+  private addAjvFormats() {
     addFormats(this._ajv, {
-      mode: 'fast',
-      formats: ['date-time']
-    })
-  }
-
-  private initLogger () {
```

Comment:
logger initialization happens in the LoggerService and is automatically injected